merp_reservation_strategy/tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright 2020 VentorTech OU
+# Part of Ventor modules. See LICENSE file for full copyright and licensing details.
+
+from . import test_stock_reservation_initial
+from . import test_stock_reservation_priority
```

Comment:
Can we name module 'stock_reservation _strategy' ? no need to have merp in name anymore.